src/solver/residuals.F90

```diff
@@ -315,7 +315,7 @@ subroutine sourceTerms_block(nn, res, iRegion, pLocal)
     use constants
     use actuatorRegionData
     use blockPointers, only : volRef, dw, w
-    use flowVarRefState, only : Pref, uRef
+    use flowVarRefState, only : pRef, uRef, LRef
```

Comment:
Because it seems this is defined as `LRef` in https://github.com/mdolab/adflow/blob/master/src/modules/flowVarRefState.F90#L43-L50 

I can change it to `lRef` if you want?

Reply:
That makes sense. `LRef` is fine.